trento/xml/article_sap_trento.xml

```diff
@@ -831,6 +831,30 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
      - what to do when database status is not green
      - what to do wen SAP instances status is not green
   -->
+   <section xml:id="sec-activity-log">
+    <title>Activity Log</title>
+    <para> Trento collects system events and user actions in the Activity Log, which can be accessed from the left hand side panel of the Trento console.</para>
```

Comment:
```suggestion
    <para> Trento collects system events and user actions in the Activity Log. It can be accessed from the left-hand side panel of the Trento console.</para>
```